meta/change_management.md

```diff
@@ -0,0 +1,68 @@
+# Change Management Process
+
+This document describes the change management process for the rule book and all related content and meta files written in Markdown in this project. The process will change based on what phase of the initial project we're in at the moment. Each phase is described below.
+
+Note that changes to this document are also governed by the process described in this document.
+
+## Phase 1: Version 1 Development
+
+The initial version development of this project consists of all work conducted on or after Sunday, November 1, 2020 through to the beginning of phase 2, as defined by phase 2. There are 4 developers involved in phase 1:
+
+- Gryphon Shafer
+- Scott Peterson
+- Jeremy Swingle
+- Zachary Tinker
+
+During the initial version development of this project, all changes will be made as follows:
+
+1. A change's author should submit the change as a *Pull Request* (PR) to the project's `master` branch
+2. At least 1 developer other than the author should comment on the PR; however, ideally many developers will comment
+3. Discussion, debate, and revisions to the PR happen all within GitHub's PR system
+4. PR authors will resolve comment threads after:
+    - When at least 1 other developer agrees with the PR author or 2 developers agree with each other, which will overrule the PR author
+    - The PR author has writen changes to the PR such that the comment thread's end-state vector is addressed
+5. The PR is merged to `master` when any of the following is true:
+    - 1 developer other than the author agrees with the change and there are no disagreements from the other developers after 1 week from PR submission
+    - 2 developers other than the author agree with the change
+
+## Phase 2: Version 1 Quality Assurance
+
+Phase 2 begins once all 4 developers agree that all primary writing and editing that was part of phase 1 has concluded. Additional editing may still take place, but at this point, all 4 developers believe they have a working product that, while potentially containing errors, is not likely missing information.
+
+The following are the steps of phase 2:
+
+1. Gryphon conducts an "OCD pass" of all content, making it align exactly with syntax, style, and other requirements
+2. Gryphon submits a PR for review by the other 3 developers
+3. Discussion and debate take place as part phase 1 PRs, and eventually the PR is merged to `master`
```

Comment:
I still subtle rewording necessary. Perhaps " 3. Discussion and debate take place as in phase 1 PRs" ?

Reply:
Cool.